docs/troubleshooting-guide/cant-log-in.md

```diff
@@ -0,0 +1,95 @@
+# People can't log in to Metabase
+
+You should be able to log in to Metabase, but:
+
+- you can't see the login page, or
+- your credentials aren't accepted.
+
+## Are you using the right URL for your Metabase?
+
+**Root cause:** the Metabase instance you are trying to log in to isn't where you think it is or isn't accessible.
+
+**Steps to take:**
+
+1. Check whether you need to include a port number as well as a hostname in the connection URL. For example, Metabase might be at `https://example.com:3000/` instead of `https://example.com/`.
+   - If you're an administrator, you'll have configured this.
+   - If you're not, please ask your admin.
+2. Check whether your Metabase instance has moved. For example, if you were using a trial instance of Metabase, but you're now in production, the URL might have changed.
```

Comment:
I don't think I have seen more than a couple having this problem over the years.